android/build.gradle

```diff
@@ -1,22 +1,27 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
     ext {
-        buildToolsVersion = "28.0.3"
+        playNameVersion = '4.0.0'
```

Comment:
All of these refactors aren't related to the PR purpose

Reply:
Copying and pasting what we wrote above:

Sorry, but is was a small change and since the PR objective touched on it I believe we don't have any problems on doing that! Any improvement, more yet when it is very small, is always welcome on any PR and in a such codebase that can leads us to forget it later.

Reply:
I'm afraid its not completely true, if any bug is introduced by some upgrade, its hard to find the reason if the upgrade is hidden by a different pr